include/zephyr/mgmt/mcumgr/grp/img_mgmt/img_mgmt.h

```diff
@@ -9,14 +9,10 @@
 #define H_IMG_MGMT_
 
 #include <inttypes.h>
-#include "img_mgmt_config.h"
-#include "image.h"
-#include "mgmt/mgmt.h"
-#include "smp/smp.h"
```

Comment:
smp.h needed for `struct smp_streamer` definition

Reply:
Base64 or not I am quite sure this is 128 characters.

Reply:
Would fail the checks, it would treat it as part of the base64 decode which would then error out. This is a binary protocol, not a text file, so should never be sent with windows-style newlines.

Reply:
Fixed with revase over #52221 merge.